libpam/include/security/_pam_macros.h

```diff
@@ -7,6 +7,8 @@
  * Organized by Cristian Gafton <gafton@redhat.com>
  */
 
+#include "config.h"
```

Comment:
Exported headers (those that are listed in `include_HEADERS`) are not allowed to `#include "config.h"` because the latter is not exported.